### PR TITLE
Towards having pyvtk on conda-forge

### PR DESCRIPTION
**Pull request by fangohr**
PyVTK is a pure Python file that allows to conveniently write VTK files to disk.

VTK stands for the Visualisation ToolKit (http://vtk.org).

### CODE REVIEW

File: recipes/pyvtk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    - pyvtk
+    - pyvtk.common
+    - pyvtk.Normals
+    - pyvtk.Scalars
+    - pyvtk.Data
+    - pyvtk.DataSet
+
+about:
+  home: https://github.com/pearu/pyvtk
+  license: NONE
```
Comment (isuruf) [inline recipes/pyvtk/meta.yaml]: According to setup.py, this is the new BSD license.

File: recipes/pyvtk/meta.yaml [hunk 2]
```diff
@@ -37,8 +37,8 @@ test:
 
 about:
   home: https://github.com/pearu/pyvtk
-  license: NONE
-  license_family: NONE
+  license: NEW BSD
+  license_family: NEW-BSD
```
Comment (isuruf) [inline recipes/pyvtk/meta.yaml]:
```
license: BSD 3-Clause
license_family: BSD
```

File: recipes/pyvtk/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+build:
+  number: 0
+  script: pip install --no-deps .
+
+requirements:
+  build:
+    - python
+    - pip
+    - numpy
```
Comment (isuruf) [inline recipes/pyvtk/meta.yaml]: I'm not sure why conda's solver is failing. Can you try adding `numpy >= 1.11`?

Comment (fangohr) [inline recipes/pyvtk/meta.yaml]: Done and pushed - let's see (I don't know why it wants to use numpy 1.9 for pyvtk.)

File: recipes/pyvtk/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+build:
+  number: 0
+  script: pip install --no-deps .
+
+requirements:
+  build:
+    - python
+    - pip
+    - numpy >= 1.11
```
Comment (isuruf) [inline recipes/pyvtk/meta.yaml]: Should be `numpy >=1.11`. No space between `=` and `1`

**Comment by fangohr** [comment]
Great - many thanks!

**Pull request merged**